### ISSUE TITLE
regression: fluvio cluster check

### ISSUE THREAD

**Issue by sehz** [infinyon/fluvio#560]
- [x] 1: Before, cluster check is used to tell how to fix but it is no longer  displaying detail which make it difficult to fix the issue (closed by #562)
- [x] 2: Check is no longer running displaying status one bye one but it is displaying only after all checks are running

```
Running pre-startup checks...
✅ ok: Kubernetes config is loadable
✅ ok: Supported kubernetes version is installed
✅ ok: Supported helm version is installed
❕ warning: Missing Fluvio system charts.
  💡 note: this may be fixed automatically during startup
✅ ok: Can create service
✅ ok: Can create customresourcedefinitions
✅ ok: Can create secret
❕ warning: Minikube tunnel not found
  💡 note: this may be fixed automatically during startup
Some checks failed, but may be auto-corrected
```

**Comment by nicholastmosher** [comment]
I can fix 1) pretty easily, when working on it I had figured that any auto-fixable thing might be better to just let the `start` command take care of it but I'll put the diagnostics back.

With regards to 2), I didn't realize that displaying the results one at a time was a requirement of the checker. The reason we were able to do that before was because we didn't have a good binary/library separation boundary. The library was just freely printing to stdout, which IMO is bad library design. I know another pattern that we can use to restore the one-by-one printout capability, but if it's not a huge deal then I'd like to push that to a future issue. Let me know what you think.
